code/nebula/neb.cpp

```diff
@@ -974,10 +987,16 @@ void neb2_render_poofs()
 		vec3d view_pos;
 		{
 			float scalar = -1 / powf((vm_vec_dist(&eye_pos, &pf.pt) / (10 * pf.radius)), 3.f);
+			if (pinfo->alignment != vmd_zero_vector)
+				scalar = 0.0f;
 
 			vm_vec_scale_add(&view_pos, &eye_pos, &eye_orient.vec.fvec, scalar);
 
 			view_pos -= pf.pt;
+
+			if (pinfo->alignment != vmd_zero_vector)
+				vm_project_point_onto_plane(&view_pos, &view_pos, &pinfo->alignment, &vmd_zero_vector);
+
```

Comment:
Is there a distinction between `pinfo->alignment` and `pf.up_vec` here?

Reply:
Yes, the `up_vec` depends on the specific poof in question, which can rotate over time on its own, specified by $Rotation. I don't really think it would look very good, but I tried to make sure rotation could at least still work in some form even if alignment is being used.

Reply:
Fair enough.  Just wanted to make sure they were both used the way they should be.